packages/amplify-e2e-core/src/init/amplifyPush.ts

```diff
@@ -2,24 +2,30 @@ import { getCLIPath, nspawn as spawn } from '..';
 
 const pushTimeoutMS = 1000 * 60 * 20; // 20 minutes;
 
+/**
+ *
```

Comment:
I wish our linter didn't create these. We should either fill in the entire jsdoc or omit this